### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by markkasaboski**
Ah, there we go. Sorry abou that.

### CODE REVIEW

File: general/classes.html [hunk 1]
```diff
@@ -38,11 +38,11 @@
 
 		<p>Unlike some other frameworks classes do not need to be loaded manually. They will be auto-loaded when you refer to them in your code (Controllers, Models, etc).</p>
 
-		<h3 id="classes_subdirectories">Classes in a sub-directory</h3>
+		<h3 id="classes_subdirectories">Classes in a Subdirectory</h3>
 
-		<p>Just like <a href="controllers/base.html">Controllers</a>, classes must be lower-case with first-letter upper case, while underscores will put the class into a sub-directory.</p>
+		<p>Just like <a href="controllers.html">controllers</a>, classes must be lower-case with first-letter upper case, while underscores will put the class into a subdirectory.</p>
```
Comment (frankdejonge) [inline general/classes.html]: This should stay the same, with /base because controllers.html does not exist.

Comment (markkasaboski) [inline general/classes.html]: Hmm, I didn't touch links.  Not sure how that happened. Will fix.

**Comment by frankdejonge** [comment]
Discussed this on IRC. Typo's and gramar things will be merged, but the Caps On The Heading must go. Classnames are first-letter-uppercased, but the rest is normal. It also depends on the context. Like "What is a Controller" should be "What is a controller", since it doesn't refer to the Controller class but to the controller concept.

**Comment by markkasaboski** [comment]
Sounds good. I'll go through and change all of the headings back and while I'm at it, I'll break out all of the shortened words like "it's" to "it is" and so on.

**Comment by markkasaboski** [comment]
Can I get a consensus on these:
Command-line or Command line
Sub-directory or Subdirectory

Each version seems to be equally accepted.  What's gonna be the Fuel way?

**Comment by frankdejonge** [comment]
lowercase subdirectory and command-line :)

**Comment by markkasaboski** [comment]
Hey Frank,

Wondering if I can get your help on this sentence:

"In half of all the methods available in the DB class new a query builder instance is returned."

From http://fuelphp.com/docs/classes/database/qb.html

It doesn't make sense at all and I'm not exactly sure how to reword it. I'm thinking that it can be omitted completely because it's not specific about which methods and it doesn't appear to be very helpful with anything anyway. 

Thoughts?

Don't worry about this one.  It's all good.

**Comment by markkasaboski** [comment]
Also, currently "Insert Builder" points to the Todo file, but qb_insert.html looks like it's ready to go. Can I uncomment it in nav.js?

**Comment by markkasaboski** [comment]
I think I just stumbled across a missing method in the mongo_Db class.  

In the mongo_db docs "http://fuelphp.com/docs/classes/mongo/methods.html" "where_like" is in the documentation but no such method exists within /core/classes/mongo/db.php. I checked within my v1 copy. 

Would you be able to have a look at that?  Should be alias for "like"? No alias either.

**Comment by frankdejonge** [comment]
Hi Clark,

I'll look into query builder description. The where_like should be just 'like'. Also I'd recommend to use separate pull-requests in managable portions. This pull-request already has conflicts and must be merged manually (resolving the conflicts).

Cheers,
Frank

**Comment by markkasaboski** [comment]
Will do. Is 20 files per pull request OK?

**Comment by markkasaboski** [comment]
OK, I'm done my editing. 

I figure it makes sense to do commits and pull requests by section. Hope that's OK.

**Comment by markkasaboski** [comment]
GitHub client for Mac wouldn't let me sync to GitHub unless all of my changes were committed.  I did them in five different commits though. Does that work for you?

File: classes/cache/config.html [hunk 2]
```diff
@@ -59,7 +59,7 @@
 						<td><pre class="php"><code>null</code></pre></td>
 						<td>
 							The default number of seconds until expiration, used when no lifetime is given. When set to
-							null it won't expire.
+							null it wo not expire.
```
Comment (philsturgeon) [inline classes/cache/config.html:62]: That's not better :)

File: classes/database/qb_where.html [hunk 3]
```diff
@@ -142,7 +142,7 @@ $query->and_where('surname', 'Doe');
 									<th><kbd>$op</kbd></th>
 									<td><em>string</em></td>
 									<td><pre class="php"><code>null</code></pre></td>
-									<td>Logic operator: =, !=, IN, BETWEEN. If none is profided this will fallback to '='</td>
```
Comment (philsturgeon) [inline classes/database/qb_where.html:145]: Wow, who wrote this bit?!

File: classes/database/qb_where.html [hunk 4]
```diff
@@ -142,7 +142,7 @@ $query->and_where('surname', 'Doe');
...
+									<td>Logic operator: =, !=, IN, BETWEEN. If none is proivided this will fallback to '='</td>
```
Comment (frankdejonge) [inline classes/database/qb_where.html:145]: proivided should be provided

File: classes/cache/config.html (hunk 2 above)
Comment (markkasaboski) [inline classes/cache/config.html:62]: Ah ya! Must have gotten pretty tired around that point.

**Comment by WanWizard** [comment]
This pull request is still open, but no response for a long time. Random checks indicate the typo's have been fixed in the meantime, so I'm closing this one.

**Pull request closed without merging**